en/peripherals/camera.md

```diff
@@ -99,6 +80,15 @@ The full list of parameters pertaining to the camera trigger module can be found
 Cameras can also (optionally) use a camera capture pin to signal the exact moment when a photo/frame is taken.
 This allows more precise mapping of images to GPS position for geotagging, or the right IMU sample for VIO synchronization, etc.
 
+Camera capture/feedback is enabled in PX4 by setting [CAM_CAP_FBACK = 1](../advanced_config/parameter_reference.md#CAM_CAP_FBACK).
```

Comment:
Is CAM_CAP_FBACK needed now you know this by the output pin being set?

Reply:
Yes the driver still needs to be enabled so that it's started.